src/Serilog.Sinks.Raygun/Sinks/Raygun/LogEventPropertyExtensions.cs

```diff
@@ -0,0 +1,30 @@
+﻿using System.Collections;
+using System.Linq;
+using Serilog.Events;
+
+namespace Serilog.Sinks.Raygun
+{
+    public static class LogEventPropertyExtensions
+    {
+        public static string AsString(this LogEventProperty property)
+        {
+            var scalar = property.Value as ScalarValue;
+            return scalar?.Value != null ? property.Value.ToString("l", null) : null;
+        }
+
+        public static int AsInteger(this LogEventProperty property, int defaultIfNull = 0)
+        {
+            var scalar = property.Value as ScalarValue;
+            return scalar?.Value != null ? int.TryParse(property.Value.ToString(), out int result) ? result : defaultIfNull : defaultIfNull;
+        }
+
+        public static IDictionary AsDictionary(this LogEventProperty property)
+        {
+            if (!(property.Value is DictionaryValue value)) return null;
+
+            return value.Elements.ToDictionary(
+                kv => kv.Key.ToString("l", null),
+                kv => (object)kv.Value);
```

Comment:
You may already have noticed this - Currently this looks like the following image in Raygun:
![image](https://user-images.githubusercontent.com/3094648/133871179-d9ab08a7-8f48-4755-ba30-95434373e4f2.png)
This is because Value here could be a ScalarValue, which has a further Value property on it. Would be nice to achieve the usual display of:
![image](https://user-images.githubusercontent.com/3094648/133871250-a51838c1-fe3c-4d67-b947-845dd6a86263.png)
Raygun4Net will send all header values as strings, so to match, here you could just ToString any ScalarValue without needing to parse numbers.

Reply:
Thanks for picking this up. I've resolved this now so the values come through as expected.